src/client/client.ts

```diff
@@ -85,7 +92,7 @@ function createDispatchers(
 ) {
   return innerActionNames.reduce((dispatchers, name) => {
     dispatchers[name] = function (...args: any[]) {
-      store.dispatch(
+      return store.dispatch(
```

Comment:
Could you explain why you’re returning the result of `dispatch` here and elsewhere? `dispatch` [returns the action that was dispatched](https://redux.js.org/api/store#dispatchaction), which I guess is not the intended return value here.

Reply:
Huh, I’m now seeing in the client test that you are getting the `transients.error` out of the move dispatcher — I’ll wait for your explanation 😄